README.md

```diff
@@ -226,6 +226,30 @@ Rails app:
     end
 ```
 
+### Server signing response
+
+The server can perform a validation of the response.
```

Comment:
I think you're trying to say the server can sign the content of the response, right?